src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,12 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        if (number < 0) {
+            return -1;
+        }
```

Comment:
let's remove this 
`In mathematics, the factorial of a positive integer n, denoted by n!, is the product of all positive integers less than or equal to n:`

Reply:
done